test/remote/gateways/remote_cecabank_test.rb

```diff
@@ -36,20 +36,19 @@ def test_successful_refund
   end
 
   def test_unsuccessful_refund
-    assert response = @gateway.refund(@amount, 'wrongreference', @options)
+    purchase = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success purchase
+
+    assert response = @gateway.refund(@amount, purchase.authorization, @options.merge(currency: 'USD'))
     assert_failure response
-    assert_equal 'ERROR', response.message
+    assert_match 'Error de comunicaciones', response.message
```

Comment:
I went with testing for the "Communications error" when sending incorrect information in the refund request. This seemed to me to be a much better alternative than testing for the "Unable to parse response" error that is received when the response is HTML (which could indicate any number of problems) caused by sending an incorrect `amount` or `authorization`.